packages/api-entrypoint/getPods.js

```diff
@@ -0,0 +1,41 @@
+import { ethers } from 'ethers'
+
+import { getPodsKey, getPoolsKey } from '../../utils/kvKeys'
+import { formatPod } from './getPod'
+
+export const getPods = async (event, request) => {
+  const _url = new URL(request.url)
+  const pathname = _url.pathname.split('.')[0]
+  const chainId = parseInt(pathname.split('/')[2], 10)
+
+  const addressesQueryParam = _url.searchParams.get('addresses')
+  let podAddresses = addressesQueryParam ? addressesQueryParam.split(',') : []
+  podAddresses = podAddresses.map((address) => address.toLowerCase())
+
+  podAddresses.map((podAddress) => {
```

Comment:
This is small but it would be more declarative to use a regular `for of`/`for in` or `forEach` here, since we're not returning an array from `map`.